common/reviews/api/node-core-library.api.md

```diff
@@ -611,7 +611,7 @@ export interface IRealNodeModulePathResolverOptions {
     // (undocumented)
     fs: Pick<typeof nodeFs, 'lstatSync' | 'readlinkSync'>;
     // (undocumented)
-    path: Pick<typeof nodePath, 'isAbsolute' | 'normalize' | 'resolve' | 'sep'>;
+    path: Pick<typeof nodePath, 'format' | 'isAbsolute' | 'parse' | 'resolve' | 'sep'>;
```

Comment:
This is a breaking change. Can we maintain the existing option as well?

Reply:
Not easily. Typical consumers won't pass the config object at all; I mostly expect this to impact the unit tests.

Reply:
Fixed.